acbs/parser.py

```diff
@@ -13,6 +13,16 @@
 generate_mode = False
 
 
+def get_defines_file_path(location: str, stage2: bool=False) -> str:
```

Comment:
Since in all the call-sites, you have the `stage2` parameter used. Let's avoid setting a default value to prevent potential logical issues in the future.

```suggestion
def get_defines_file_path(location: str, stage2: bool) -> str:
```